PUBLISHING.md

```diff
@@ -4,11 +4,19 @@
 
 This document outlines the process for publishing the Smartsheet Live Data Connector User Guide onto Github Pages. The actual Smartsheet Live Data Connector documentation can be viewed [here](http://smartsheet-platform.github.io/odbc-docs). 
 
+## Why git version tags?
+Git tags are a nice way to keep track of the versions of the documentation. The version of the documentation should match the version of the Smartsheet Live Data Connector that it supports.
```

Comment:
maybe add an example of how to git tag and explanation on how to use semantic versioning

ex. `git tag v1.0.1 after a patch`

Reply:
Agree. Add this syntax example.